shell/platform/common/accessibility_bridge.h

```diff
@@ -39,6 +41,7 @@ namespace flutter {
 class AccessibilityBridge
     : public std::enable_shared_from_this<AccessibilityBridge>,
       public FlutterPlatformNodeDelegate::OwnerBridge,
+      public ui::AXPlatformTreeManager,
```

Comment:
For multi-view work, we will need a `AXTree` for each view. @dkwingsmt's plan was to make each view own their own `AccessibilityBridge`. From my understanding, there should only be a single `AXPlatfromTreeManager` for all `AXTree`s. Is that correct? Does that mean that having the view own its own `AccessibilityBridge` won't be a viable solution going forward?

Reply:
The usage of the class suggests that there should be an instance of `AXPlatformTreeManager` for each `AXTreeID` that is used. If each view owns an `AccessibilityBridge` and has its own `AXTree`, this sounds consistent with its use.

Reply:
Gotcha, thanks!